src/driver/mode_initial.rs

```diff
@@ -5,12 +5,11 @@ use display_interface::WriteOnlyDataCommand;
 use super::{mode_graphics::GraphicsMode, mode_raw::RawMode};
 use crate::{DisplaySpecs, GraphicsPageBuffer, ST7565};
 
+/// This mode is purely to transition into other modes.
 pub struct InitialMode;
 
 /// ---- Functionality of the initial mode ----
 /// ===========================================
-///
-/// This mode is purely to transition into other modes.
```

Comment:
I think I like how the documentation looked previously, so I'd prefer not to delete those two lines. I don't mind the duplication in the docstring of `InitialMode`.